src/Playwright.Tests/BaseTests/HttpService.cs

```diff
@@ -38,10 +39,11 @@ public static Task<HttpService> Register(WorkerAwareTest test)
             var workerIndex = test.WorkerIndex;
             return test.RegisterService("Http", async () =>
             {
+                var assetDir = Path.Combine(TestUtils.FindParentDirectory("Playwright.Tests.TestServer"), "assets");
                 var http = new HttpService
                 {
-                    Server = SimpleServer.Create(8907 + workerIndex * 2, TestUtils.FindParentDirectory("Playwright.Tests.TestServer")),
```

Comment:
the accessDir was previously not used but still passed, now it gets used (passed externally, gets stored as a property inside the class and used inside ServeFile).